### PR TITLE
feat: body scroll was blocked when location dropdown is visible

### PR DESCRIPTION
**Pull request by roger-in-kiva**

### CODE REVIEW

File: src/components/LoansByCategory/QuickFilters/LocationSelector.vue [hunk 1]
```diff
@@ -206,6 +206,11 @@ export default {
 		toggleRegions() {
 			this.showRegions = !this.showRegions;
 			this.selectedRegion = null;
+			if (this.showRegions) {
+				document.documentElement.style.overflow = 'hidden';
```
Comment (dyersituations) [inline src/components/LoansByCategory/QuickFilters/LocationSelector.vue:210]: Usually you don't want to change the document directly in Vue. Does this dropdown behave different than other dropdowns we have?

Comment (roger-in-kiva) [reply src/components/LoansByCategory/QuickFilters/LocationSelector.vue:210]: Yeah, behavior is diff for mobile, need to block the scroll of the body and allow it into the modal.

Comment (dyersituations) [reply src/components/LoansByCategory/QuickFilters/LocationSelector.vue:210]: I guess I'm confused as to what this is solving exactly. Is there a reason this dropdown in particular needs a scrolling fix? Do we do this for other dropdowns? I didn't see this code anywhere else.

Comment (roger-in-kiva) [reply src/components/LoansByCategory/QuickFilters/LocationSelector.vue:210]: Yeah context is here, 0:26 sec aprox

https://kivahq.slack.com/files/U016NHS0FM5/F0479EMGQM7/screenshare_-_2022-10-17_2_45_37_pm.webm?origin_team=TAY80LL2D&origin_channel=D03FN1M1KDY

Comment (mcstover) [reply src/components/LoansByCategory/QuickFilters/LocationSelector.vue:210]: In the past, we've created a full screen lightbox for filtering on mobile which could be how we address this in a future, full implementation as well.

Comment (roger-in-kiva) [reply src/components/LoansByCategory/QuickFilters/LocationSelector.vue:210]: Yeah, there is an overlay in design similar to having a lightbox, but is out of the scope for the experiment.
If this becomes a feature we'll be doing all needed refactor for a full implementation.

Comment (dyersituations) [reply src/components/LoansByCategory/QuickFilters/LocationSelector.vue:210]: Thanks for the context.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**